api/Parser.cpp

```diff
@@ -381,6 +381,13 @@ EndpointDescription parsePatchEndpoint(const nlohmann::json& data, const std::st
         endpointDescription.data.set(dataChannel);
     }
 
+    if (data.find("neighbours") != data.end())
+    {
+        for (auto& group : data["neighbours"]["groups"])
```

Comment:
Protect against SEGV on BadRequests

```suggestion
        for (auto& group : requiredJsonArray(data["neighbours"]), "groups"))
```